lib/dRep/dRepImpl.py

```diff
@@ -1,24 +1,42 @@
 # -*- coding: utf-8 -*-
 #BEGIN_HEADER
+####################################################################################################
+#---------------------------------------------------------------------------------------------------
+#---------------------------------------------------------------------------------------------------
+#---------------------------------------------------------------------------------------------------
+
+
 import logging
 import os
+import sys
 import shutil
 import subprocess
 import pprint
 import uuid
-
+import re
+import functools
+import pickle
+import warnings
 
 from installed_clients.KBaseReportClient import KBaseReport
 from installed_clients.DataFileUtilClient import DataFileUtil
 from installed_clients.WorkspaceClient import Workspace
 from installed_clients.MetagenomeUtilsClient import MetagenomeUtils
 
-
-
-from .util import *
+from .util import PrintUtil, KBaseObjUtil, OutputUtil
 from .util.PrintUtil import *
 from .util.KBaseObjUtil import *
 
+subprocess.run = functools.partial(subprocess.run, shell=True, stdout=subprocess.PIPE, 
```

Comment:
Rather than overwriting this, I would assign it to your own function name like "run_subprocess". Otherwise this might be really hard for others to figure out when they call it and they expect the api to be `subprocess.run`

Reply:
good idea